TSG.md

```diff
@@ -0,0 +1,207 @@
+# Common Mount Problems
+**Logging**
+
+Please ensure logging is turned on DEBUG mode when trying to reproduce an issue.
+This can help in many instances understand what the underlying issue is.
+
+A useful setting in your configuration file to utilize when debugging is `sdk-trace: true` under the azstorager component. This will log all outgoing REST calls.
```

Comment:
Typo
```suggestion
A useful setting in your configuration file to utilize when debugging is `sdk-trace: true` under the azstorage component. This will log all outgoing REST calls.
```